src/jaxoplanet/orbits/keplerian.py

```diff
@@ -400,7 +400,7 @@ def __init__(
             self.time_transit = jnpu.zeros_like(self.period)
 
         if map is None:
-            self.map = Map(y=Ylm({(0, 0): 0, (1, 0): 0.0}))
+            self.map = Map(amplitude=0.0)
```

Comment:
This is a place where I think it would be good to allow a `None` map to indicate a system where there is no map because then we can speed up the computations rather than computing a light curve and then multiplying by zero!

Reply:
I am preparing a separate PR where I handle all the `None` arguments in the `starry.light_curve` module (see [feat/none-args-lightcurves](https://github.com/lgrcia/jaxoplanet/tree/feat/none-args-lightcurves))

Reply:
Great!